packages/salesforcedx-vscode-lwc/test/extension.test.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+
+describe('LWC Extension Tests', () => {
```

Comment:
This is just a sample test for now to appease the code coverage collection routine.